Controllers/controllers.errors.js

```diff
@@ -1,6 +1,16 @@
 exports.handle404Paths = (req, res, next) => {
   res.status(404).send({ msg: "Page not found!" });
 };
+exports.handleCustomPaths = (err, req, res, next) => {
+  if (err.msg !== undefined) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+};
+exports.handle400Paths = (err, req, res, next) => {
+  res.status(400).send({ msg: "Bad request - invalid id" });
```

Comment:
Remember that you still need to be able to pass errors to the 500 error handler if they aren't 400 errors - you still need a If Else situation in handle400Paths